WordPress/Classes/Models/Post.swift

```diff
@@ -204,6 +242,15 @@ class Post: AbstractPost {
 
                 return true
             }
+
+            if publicizeMessage ?? "" != originalPost.publicizeMessage ?? "" {
```

Comment:
We want the default values there to avoid detecting changes for nil vs the empty string.